docs/jupyterWidgets/usage.rst

```diff
@@ -90,3 +86,19 @@ This example shows how to visualize features through a Jupyter widget.
 
 .. image:: ../img/jupyterWidgets/featureWidget.png
 
+
+Genotype Example
+---------------
+
+This example shows how to visualize genotypes through a Jupyter widget.
+
+.. code:: python
+
+    # make genotype track
+    tracks=[Track(viz="features", label="my Features", source=pileup.sources.VcfDataSource(./data/genodata.v3.vcf))]
```

Comment:
see comment above. Also, I dont think this would work because the filepath is not in quotes

Reply:
This is a feature visualization, not genotypes

Reply:
will fix this... was an oversight